packages/alt-ergo/alt-ergo.1.30/opam

```diff
@@ -0,0 +1,39 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
+
+opam-version: "1.2"
+version: "1.30"
+
+maintainer: "alt-ergo@ocamlpro.com"
+authors: "Alt-Ergo developers"
+license: "CeCILL-C"
+homepage: "http://alt-ergo.ocamlpro.com/"
+bug-reports: "https://github.com/OCamlPro/alt-ergo-private/issues"
```

Comment:
Ah yes! because I just copied the internal OPAM file. I should replace alt-ergo-private with alt-ergo